Annotation/alternative_golden_set.json

```diff
@@ -912,7 +924,7 @@
                     }

```

Comment:
They're not ECs (we're ignoring nominal complements altogether). But it's weird because I can't read this sentence in any way that sounds grammatical. So I just ignored the determiner and treated it as a verb